test/_requestToken.js

```diff
@@ -71,7 +71,7 @@ describe('_requestToken', function() {
 	it('should fail if request module fails', function(done) {
 		// subsequently, this will test merging options as well
 		AuthClient
-			._requestToken({ method: 'TEST' })
```

Comment:
Well, `nock` only sets up the mock for the next request. This test will not have any `nock` set up and so any call would fail, which is why they're passing. I can add it back, but I was talking with @nathan-boyd and it seemed weird that this would call a weird method as a test.